scripts/collect_ad_boilerplate.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import re
+
+import logging
+
+
+def write_detector_class(boilerplate_file, dev_name, det_name, cam_name):
+    """
+    Writes boilerplater 'Detector' class for ophyd/areadetector/detectors
```

Comment:
Can the explanation of parameters be added here? I am having trouble immediately distinguish between `dev_name` and `det_name`.

Reply:
Done, thanks.